graphqlapi/src/main/java/io/stargate/graphql/schema/DdlSchemaBuilder.java

```diff
@@ -453,6 +457,41 @@ private GraphQLFieldDefinition buildCreateTable() {
         .build();
   }
 
+  private GraphQLFieldDefinition buildCreateIndex() {
+    return GraphQLFieldDefinition.newFieldDefinition()
+        .name("createIndex")
+        .argument(GraphQLArgument.newArgument().name("keyspaceName").type(Scalars.GraphQLString))
+        .argument(
+            GraphQLArgument.newArgument().name("tableName").type(nonNull(Scalars.GraphQLString)))
+        .argument(
+            GraphQLArgument.newArgument().name("columnName").type(nonNull(Scalars.GraphQLString)))
+        .argument(GraphQLArgument.newArgument().name("indexName").type(Scalars.GraphQLString))
+        .argument(
+            GraphQLArgument.newArgument().name("customIndexClass").type(Scalars.GraphQLString))
+        .argument(GraphQLArgument.newArgument().name("ifNotExists").type(Scalars.GraphQLBoolean))
+        .argument(GraphQLArgument.newArgument().name("indexEntries").type(Scalars.GraphQLBoolean))
+        .argument(GraphQLArgument.newArgument().name("indexKeys").type(Scalars.GraphQLBoolean))
+        .argument(GraphQLArgument.newArgument().name("indexValues").type(Scalars.GraphQLBoolean))
+        .argument(GraphQLArgument.newArgument().name("indexFull").type(Scalars.GraphQLBoolean))
```

Comment:
Might be good to surface these as descriptions

https://github.com/stargate/stargate/blob/master/persistence-api/src/main/java/io/stargate/db/query/builder/QueryBuilderImpl.java#L1049-L1062